reana/cli.py

```diff
@@ -556,13 +782,26 @@ def docker_push(user, tag, component):
 @click.option('--component', '-c', multiple=True, default=['CLUSTER'],
               help='Which components? [name|CLUSTER]')
 @cli.command(name='docker-pull')
-def docker_pull(user, tag, component):
+def docker_pull(user, tag, component):  # noqa: D301
     """Pull REANA component images from DockerHub.
 
-    The option ``component`` can be repeated and the value can be a name such
-    as 'reana-job-controller', or a special value 'CLUSTER' meaning all
-    cluster-related components, or special value 'ALL' meaning all REANA
-    repositories. [default=CLUSTER]
+    \b
+    :param components: The option ``component`` can be repeated. The value may
+                       consist of:
+                         * (1) standard component name such as
+                               'reana-job-controller';
+                         * (2) short component name such as 'r-j-controller';
+                         * (3) special value '.' indicating component of the
+                               current working directory;
+                         * (4) special value 'CLUSTER' that will expand to
+                               cover all REANA cluster components [default];
+                         * (5) special value 'ALL' that will expand to include
+                               all REANA repositories.
+    :param user: DockerHub organisation or user name. [default=reanahub]
+    :param tag: Docker tag to use. [default=latest]
+    :type component: str
+    :type user: str
+    :type tag: str
```

Comment:
Line 808 `if` statement is wrong, tests that if it is not dockerised then execute Docker command. I didn't spot it in previous review 😞, could you please change it? :)